expat/lib/xmlparse.c

```diff
@@ -6118,18 +6168,18 @@ storeEntityValue(XML_Parser parser, const ENCODING *enc,
         = entityTextPtr; /* XmlEntityValueTok doesn't always set the last arg */
     int tok = XmlEntityValueTok(enc, entityTextPtr, entityTextEnd, &next);
 
-#ifdef XML_DTD
+#  if XML_GE == 1
```

Comment:
Redundant check -- the whole function exists only when XML_GE == 1.

Reply:
Confirming, will be gone with the next push in a few minutes. Thanks!